.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+esy_cache_key: &esy_cache_key
+  key: esy-cache-{{ checksum "esy.json" }}-{{ checksum "esy.lock" }}-{{ checksum ".circleci/config.yml" }}
+
+version: 2
+jobs:
+  build:
+    working_directory: ~/repo
+    docker:
+      - image: circleci/node:7.10
+    steps:
+      - checkout
+      - run: sudo npm -g install esy@preview
+      - restore_cache:
+          <<: *esy_cache_key
+      - run: esy install
+      - save_cache:
+          <<: *esy_cache_key
+          paths:
+            - ~/.esy
+      - run: esy build
+      - run: yarn test
```

Comment:
let's not make this detour through yarn. We should just run `esy jbuilder runtest --no-buffer` or have a `./test` script that does just that.